src/transform/FooterLegal.js

```diff
@@ -6,16 +6,23 @@ import './FooterLegal.css'
  */
 
 /**
+  * @typedef {function} FooterBrowserClickCallback
+  * @return {void}
+  */
+
+/**
  * Adds legal footer html to 'containerID' element.
  * @param {!Element} content
  * @param {?string} licenseString
  * @param {?string} licenseSubstitutionString
  * @param {!string} containerID
  * @param {!FooterLegalClickCallback} licenseLinkClickHandler
+ * @param {?string} viewInBrowserString
```

Comment:
Should this  be nonnull? When `viewInBrowserString` is `undefined`, `` `foo${viewInBrowserString}bar` === "fooundefinedbar"`` and the callback must be nonnull which is a little odd for an empty link.